tutorials/migrating/upgrading_to_godot_4.rst

```diff
@@ -412,8 +412,12 @@ table to find its new name.
 - BaseButton's ``set_event()`` is now ``set_shortcut()``.
 - Camera2D's ``get_v_offset()`` is now ``get_drag_vertical_offset()``.
 - Camera2D's ``set_v_offset()`` is now ``set_drag_vertical_offset()``.
+- Camera2D's ``make_current()`` is now ``set_current()``.
```

Comment:
[This still exists](https://docs.godotengine.org/en/stable/classes/class_camera2d.html#class-camera2d-method-make-current)